exercises/difference-of-squares/src/test/java/DifferenceOfSquaresCalculatorTest.java

```diff
@@ -4,10 +4,7 @@
 
 import static org.junit.Assert.assertEquals;
 
-/*
- * version: 1.1.0
- */
-public final class DifferenceOfSquaresCalculatorTest {
+public class DifferenceOfSquaresCalculatorTest {
```

Comment:
Yes, there was a conflict that needed to be resolved and it wasn't in the original file so I think I got rid of it.